datadog_checks_dev/datadog_checks/dev/tooling/commands/config.py

```diff
@@ -82,6 +82,14 @@ def set_value(ctx, key, value):
     New setting:
     [github]
     user = "foo"
+
+    You can also assign values on a per-org basis.
+
+    \b
```

Comment:
Could this also describe how you need to set an org?

Reply:
Do you mean how to set an org in datadoghq or why we need orgs?

Reply:
like`ddev config set org hq`

Reply:
Might be worth an internal wiki page at that point tbh. There's a few things to describe: how to add orgs to the config, what fields you can put here, how to use switch from one org to another (either using the config or the cli flag)

Reply:
True, that would be helpful!